wiki/Guides/BSS_Issues/de.md

```diff
@@ -0,0 +1,40 @@
+---
+tags:
+  - I can't submit my own beatmap!
+  - beatmap submission system issues
+  - Ich kann meine eigenen Beatmaps nicht einreichen!
+  - Probleme mit dem Beatmap Submission System
+---
+
+# Wie man Probleme mit dem BSS löst
+
+Wenn das **[Beatmap Submission System](/wiki/Submission)** (***BSS***) dich nicht deine eigenen Beatmaps einreichen lässt, weil es der Meinung ist, dass die Beatmap nicht dir gehört, dann mache folgendes:
+
+1. Gehe zum Ordner der Beatmap.
+   - Du kannst es leicht erreichen, indem du `Datei` und dann `Songs-Ordner öffnen` im [Beatmap-Editor](/wiki/Client/Beatmap_editor) anklickst.
```

Comment:
```suggestion
   - Du kannst ihn leicht erreichen, indem du `Datei` und dann `Songs-Ordner öffnen` im [Beatmap-Editor](/wiki/Client/Beatmap_editor) anklickst.
```

Alternatively `Das kannst du leicht erreichen`, if that is what you were going for.